packages/mesos/buildinfo.json

```diff
@@ -3,7 +3,7 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/mesosphere/mesos",
-    "ref": "a22b5dd552168a8dd925682f3b349059e10fc3d3",
+    "ref": "b546e7544c04a6d3e0da87cd6207ae8f4d827add",
```

Comment:
I've been watching the Mesos SHA closely. This SHA is a proper descendant of what's in master of dcos/dcos, since we only linearly progress the `dcos-mesos-1.0.1` branch, which is official Apache Mesos `1.0.1` tag plus a series of patches. So this part of the PR would be acceptable to merge now.
However, if #768 lands first, then it supplants (and includes) these changes to the dcos-mesos branch, since it includes all of the cherry-picks from `dcos-mesos-1.0.1` at Joseph's new sha (b546e75), but rebased on top of the Apache Mesos `1.0.x` branch, which includes `1.0.1` exactly, plus cherry-picks from master that are destined to be released in `1.0.2` whenever that is cut.